slides/reveal-js/technical_introduction_20240912/index.html

```diff
@@ -0,0 +1,343 @@
+                            schnell
+                            zu identifizieren.</li>
+                    </ul>
+                </div>
+            </section>
+
+            <section>
+
+                <section>
+                    <h3>Wie funktioniert das ATF?</h3>
+                    <div style="font-size:0.75em; margin-top:0px; flex: 1; margin-left: 20px;">
+                        <p><strong>Kernfunktionen und Prozesse:</strong></p>
+                        <p>Das ATF nutzt standardisierte Prozesse für die Systemkommunikation, die speziell auf einzelne
+                            Anwendungsfälle zugeschnitten sind:</p>
+                        <ul>
+                            <li><strong>Anwendungsfall-Repräsentation:</strong> Jeder <a
+                                    href="https://simplifier.net/app-transport-framework/app-tags-cs"
+                                    target="_blank">Anwendungsfall wird durch ein CodeSystem</a> repräsentiert, um
+                                Eindeutigkeit in Identifizierung und Verarbeitung sicherzustellen.</li>
+                            <li><strong>Verzeichnisdienst (VZD):</strong> Im VZD wird hinterlegt, welche KIM-Adressen
+                                das ATF unterstützen.</li>
+                            <li><strong>Capability-Message:</strong> Der Sender einer Nachricht kann automatisiert beim
+                                Empfänger nachfragen, ob der gewünschte Use Case unterstützt wird.</li>
+                            <li><strong>Nachrichten-Repräsentation:</strong> Jede <a
+                                    href="https://simplifier.net/app-transport-framework/service-identifier-cs"
+                                    target="_blank">ATF-Nachricht verwendet ein CodeSystem</a>, um ihren Anwendungsfall
+                                und Zweck zu klären.</li>
+                            <li><strong>Kapselung der Anwendungsfalldaten:</strong> Das ATF fasst die FHIR-Ressourcen
+                                eines Anwendungsfalls in einer übergeordneten Ressource (FHIR Bundle) zusammen und
+                                integriert relevante Metadaten im Kopfbereich dieser Ressource.</li>
+                            <li><strong>Empfangsbestätigungen:</strong> Jede Nachricht erhält automatisch eine
+                                standardisierte Bestätigung, die über die erfolgreiche Verarbeitung oder Fehler
+                                informiert und so die Sicherheit und Qualität der Kommunikation erhöht.</li>
+                            <li><strong>Zuordenbarkeit:</strong> Jede Nachricht kann einer sendenden Software und einem
+                                Hersteller zugeordnet werden, der über eine angegebene Adresse kontaktiert werden kann.
+                            </li>
+                            <li><strong>Fallback-Szenario:</strong> Für jede Nachricht eines Use Cases kann eine HTML-
```

Comment:
Vorschlag:
Jeder Nachrichtentyp kann über eine XSLT-Definition in ein HTML überführt werden, um den Nachrichteninhalt menschenlesbar zu machen. Der Anhang wird neben dem FHIR-Datensatz bereitgestellt.